examples/player/simid_player.html

```diff
@@ -196,7 +196,9 @@
         <button onclick="switchCreative('selector.html')">selector</button>
         <button onclick="switchCreative('extender.html')">extender</button>
         <button onclick="switchCreative('testers_nonlinear.html')">testers_nonlinear</button>
+        <button onclick="switchCreative('banner_nonlinear.html')">banner_nonlinear</button>
```

Comment:
When you switch to this creative, it would be good to switch the radio button to nonlinear.  You could also set the default duration for this to 25 seconds (or something like that).

Reply:
Can we handle this in a future pr?